### DISCUSSION TITLE
Welcome to BrailleRap Discussions!

### DISCUSSION THREAD

**Discussion by braillerap** [braillerap/BrailleRap#2] [Announcements]
## 👋 Welcome!
  We’re giving a try to  Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WallK** [comment]
Hello! Thanks for your great work creating and supporting such a useful project!
I was curious, what do you think about 6 dot "head"? I mean, something than can punch the whole cell at once
Is this project more about graphics so it's not feasible? Is this idea just not DIY friendly?
I understand that this will be complicated mechanically, but could speed up printing a lot!
Thank you!

**Reply by crocsg** [reply]
Hello
Something that punch the whole cell at once is something that come in mind. But not so simple as saying it, Braille dimensions are precises, and you also need regularity in Braille dots.
Obviously this can speed up the device by 6. But do we need fastest device ? Not at the cost of Braille quality in regularity (Braille stylus are hand made)

**Reply by WallK** [reply]
I was thinking that rigid cell with 6 styluses is much more repeatable than to move point of impact 5 times
But you have a great point in hard repeatability of handmade punches
